<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fottoshoppi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>761</width>
      <height>891</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="vert_mirror">
       <property name="text">
        <string>Mirror Vertically</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hor_mirror">
       <property name="text">
        <string>Mirror Horizontally</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="grayscale_button">
       <property name="text">
        <string>Grayscale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="negativeButton">
       <property name="text">
        <string>Negative</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="histogramButton">
       <property name="text">
        <string>Histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="equalizeButton">
       <property name="text">
        <string>Equalize histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="matchHistogramButton">
       <property name="text">
        <string>Match histogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomInButton">
       <property name="text">
        <string>Zoom in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="rotateLeftButton">
         <property name="text">
          <string>Rotate left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotateRightButton">
         <property name="text">
          <string>Rotate right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="contrastButton">
         <property name="text">
          <string>Apply contrast</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="contrast">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="addBrightnessButton">
         <property name="text">
          <string>Add brightness</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="brightness">
         <property name="minimum">
          <number>-255</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="quantize_button">
         <property name="text">
          <string>Quantize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="quantize_tones">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="zoomOutButton">
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Zoom out (X, Y)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="zoomOutX"/>
       </item>
       <item>
        <widget class="QSpinBox" name="zoomOutY"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="conv0">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.062500000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDoubleSpinBox" name="conv3">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.125000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="conv8">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.062500000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="conv2">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.062500000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="conv7">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.125000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="conv4">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.250000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QComboBox" name="convEffect">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Gaussian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Laplacian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High pass</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prewitt Hx</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prewitt Hy</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sobel Hx</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sobel Hy</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="conv5">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.125000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDoubleSpinBox" name="conv6">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.062500000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="conv1">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="minimum">
          <double>-100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>0.125000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QPushButton" name="convolveButton">
         <property name="text">
          <string>Convolve</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
